src/util/helpers.py

```diff
@@ -15,6 +15,24 @@ def find_game_elements(table):
     # See https://www.crummy.com/software/BeautifulSoup/bs4/doc/#searching-by-css-class for syntax here
     return table.find_all('tr', class_=re.compile('TrGame*'))
 
+# Replace text 'half' with '.5
+def clean_line(text):
+    return text.replace('½','.5')
+
+# Extracts line or odds from string Ex: '-3 -120'
+def convert_line(text, odds = False):
+    length = len(text)
+    if odds:
+        return text[length - 4:].strip()
+    else:
+        return text[:length - 4].strip()
+
```

Comment:
rather than using a magic number (4), what if we split the string by whitespace?

something like:

```suggestion
def convert_line(text, odds = False):
    if !odds:
        return text.split(' ')[0]
    return text.split(' ')[1]

```
more on magic #s: https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

Reply:
For the dog website there isn't whitespace inbetween the line and the odds ie '-3-110' vs '-3 -110'. Thats why the number is 4